### ISSUE TITLE
Readline

### ISSUE THREAD

**Issue by kalbasit** [dtannen/monitor#6]
What's the purpose of storing readline in the repository?

**Comment by dtannen** [comment]
Yes, when I have a chance I will remove it and change the README, etc.
